### DISCUSSION TITLE
400 passing device_id list

### DISCUSSION THREAD

**Discussion by jshcodes** [CrowdStrike/falconpy#895] [Q&A] [answered]
> Asked by @brucelourenco

Hello guys!
I have an issue near to issue #525, however, I'm facing error 400 as follows:

![image](https://user-images.githubusercontent.com/45561175/214596169-43c03e8b-7cea-40d1-a868-0c97f61e890f.png)

If I don't use f-strings and pass the device ids directly works fine

Follow falconpy version:
Name: crowdstrike-falconpy
Version: 1.2.10
Summary: The CrowdStrike Falcon SDK for Python 3
Home-page: https://github.com/CrowdStrike/falconpy
Author: CrowdStrike
Author-email: falconpy@crowdstrike.com
License:
Location: /root/miniconda3/lib/python3.9/site-packages
Requires: urllib3, requests
Required-by:

What I'm doing wrong?

Thank you everyone!

_Originally posted by @brucelourenco in https://github.com/CrowdStrike/falconpy/discussions/525#discussioncomment-4778485_

**Comment by jshcodes** [comment, marked as answer]
Hi @brucelourenco!

You're almost there, the filter string is using the wrong parameter name. (Line 18, `device_list` should be `device_id`.)

Change your `act_params` list of dictionaries to the following and try it again?

```json
[{
    "name": "filter",
    "value": "(device_id:{device_list})"
}]
```

**Reply by brucelourenco** [reply]
Oh dude! I can believe I did this lol. 
I need to pay more attention before posting.
Thanks a lot @jshcodes !!
PS: It works perfectly now lol

**Reply by jshcodes** [reply]
W00t!! Happy to assist! 😀
